sweep/tx_input_set.go

```diff
@@ -141,22 +141,28 @@ func validateInputs(inputs []SweeperInput, deadlineHeight int32) error {
 	// dedupInputs is a set used to track unique outpoints of the inputs.
 	dedupInputs := fn.NewSet(
 		// Iterate all the inputs and map the function.
-		fn.Map(func(inp SweeperInput) wire.OutPoint {
-			// If the input has a deadline height, we'll check if
-			// it's the same as the specified.
-			inp.params.DeadlineHeight.WhenSome(func(h int32) {
-				// Exit early if the deadlines matched.
-				if h == deadlineHeight {
-					return
-				}
-
-				// Update the deadline height if it's
-				// different.
-				inputDeadline = h
-			})
-
-			return inp.OutPoint()
-		}, inputs)...,
+		fn.Map(
```

Comment:
I'm glad we changed the semantics before we're using `fn` everywhere. That diff wasn't too hard to get through :+1: